### PR TITLE
cleanup report, and puppet supported regex

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

**Comment by tphoney** [comment]
@mhaskel can you please have a look

File: stats.rb [hunk 1]
```diff
@@ -134,17 +138,19 @@ def tablecreation(title,pr_array)
...
 html.push(htmlchunk)
 htmlchunk = tablecreation("PRs that have yet to be commented on by a puppet member:",array_puppet_uncommented_pulls)
 html.push(htmlchunk)
-htmlchunk = tablecreation("PRs that community have asked for help:",array_mentioned_pulls)
+htmlchunk = tablecreation("PRs that community have asked for help (mentioned a puppet member):",array_mentioned_pulls)
+html.push(htmlchunk)
+htmlchunk = tablecreation("PRs that require rebase (needs comment and a label):",array_needs_rebase_pulls)
 html.push(htmlchunk)
-htmlchunk = tablecreation("PRs that require rebase:",array_needs_rebase_pulls)
+htmlchunk = tablecreation("PRs that should die, no activity for 40 days:",array_needs_rebase_pulls)
```
Comment (underscorgan) [inline stats.rb]: Can this be reworded to 'should be closed' or something less harsh than 'should die'?

**Comment by tphoney** [comment]
@mhaskel apologies, cleaned up

**Comment by underscorgan** [comment]
@tphoney thanks! Could you also squash/rebase?

**Comment by tphoney** [comment]
apologies @mhaskel i mashed the keyboard badly :(

**Comment by underscorgan** [comment]
thanks @tphoney !

**Pull request merged**